### PR TITLE
fix tools/build/setnextversion.sh: do not update tools/apidoc/pom.xml

### PR DESCRIPTION
**Pull request by weizhouapache**
Steps to reproduce the issue

(1) on ubuntu server, run "git checkout 4.17.2.0"
(2) packaging
cd packaging/
./build-deb.sh -b test1

expected result: succeed
actual result:

[ERROR] The build could not read 1 project -> [Help 1] [ERROR]
[ERROR]   The project org.apache.cloudstack:cloud-apidoc:4.17.2.0-test1-test1 (/data/cloudstack/tools/apidoc/pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM for org.apache.cloudstack:cloud-apidoc:4.17.2.0-test1-test1: Could not find artifact org.apache.cloudstack:cloud-tools:pom:4.17.2.0-test1-test1 in central (https://repo.maven.apache.org/maven2) and 'parent.relativePath' points at wrong local POM @ line 25, column 13 -> [Help 2]

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [x] Minor
- [ ] Trivial

### CODE REVIEW

File: tools/build/setnextversion.sh [hunk 1]
```diff
@@ -131,7 +131,6 @@ echo "setting new version numbers"
 mvn versions:set -DnewVersion=$version -P vmware -P developer -P systemvm -P simulator -Dnoredist versions:commit
 
 perl -pi -e "s/$currentversion/$version/" debian/changelog
-perl -pi -e "s/$currentversion/$version/" tools/apidoc/pom.xml
```
Comment (rohityadavcloud) [inline tools/build/setnextversion.sh:134]: @weizhouapache there's no apidoc pom.xml? or data in it to sed/grep (btw sed would be faster than perl, but this is good too)

Comment (weizhouapache) [reply tools/build/setnextversion.sh:134]: @rohityadavcloud 
the version number in tools/apidoc/pom.xml has already been updated by line 131
```
mvn versions:set -DnewVersion=$version -P vmware -P developer -P systemvm -P simulator -Dnoredist versions:commit
```

This fixes the issue that debian packages cannot be build in private repository.

**Review by DaanHoogland — APPROVED** [review]
builds and looks good

**Comment by DaanHoogland** [comment]
@wido did you guys face this as well?
@rohityadavcloud do you approve?

**Review by rohityadavcloud — APPROVED** [review]

**Comment by rohityadavcloud** [comment]
@shwstppr can you review/test?

**Comment by rohityadavcloud** [comment]
ping @shwstppr can you verify/test and help merge it? Thanks.

**Review by shwstppr — APPROVED** [review]
LGTM. Tested on Ubuntu 20.04 with build script

**Pull request merged**